docs/architecture/blazor-for-web-forms-developers/introduction.md

```diff
@@ -8,6 +8,53 @@ ms.date: 09/11/2019
 
 # An introduction to Blazor for ASP.NET Web Forms developers
 
-ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. 
+ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. Back when the Web was still largely in its infancy, ASP.NET Web Forms made building web apps simple and productive by adopting many of the patterns that were used for desktop development. In ASP.NET Web Forms apps, web pages are composed from reusable UI controls. User interactions are handled as events. A rich ecosystem of Web Forms UI controls provided by Microsoft and third-party control vendors makes connecting to data sources and displaying rich visualizations easy. For the visually inclined, the Web Forms designer provides a simple drag-and-drop interface for laying out controls.
```

Comment:
```suggestion
ASP.NET Web Forms has been a staple of .NET web development since the .NET Framework first shipped in 2002. Back when the Web was still largely in its infancy, ASP.NET Web Forms made building web apps simple and productive by adopting many of the patterns that were used for desktop development. In ASP.NET Web Forms apps, web pages are composed from reusable UI controls. User interactions are handled as events. A rich ecosystem of Web Forms UI controls provided by Microsoft and third-party control vendors makes connecting to data sources and displaying rich visualizations easy. For the visually inclined, the Web Forms designer provides a simple drag-and-drop interface for managing controls.
```